pkg/plugin/v3/webhook.go

```diff
@@ -76,6 +78,8 @@ func (p *createWebhookPlugin) BindFlags(fs *pflag.FlagSet) {
 		"if set, scaffold the validating webhook")
 	fs.BoolVar(&p.conversion, "conversion", false,
 		"if set, scaffold the conversion webhook")
+	fs.StringVar(&p.spoke, "spoke", "", "provide a hub version for the conversion webhook")
+	fs.StringVar(&p.hub, "hub", "", "provide the non-hub (spoke) verision for conversion webhook")
```

Comment:
Can we simplify these flags by removing `--conversion`, and treating setting `conversion: true` as `--hub --spoke`, seeing as:
- `--hub --spoke` always requires `--conversion` always be set
- there's no point in setting `--conversion` without `--hub --spoke`

Reply:
Since it was suggested that we preserve the current implementation, have modified this such that we can either specify `hub` and `spoke`(need not add `--conversion`) or just add `--conversion`. If former format is followed, we would scaffold implementations for hub & spoke, if not then do the same as it was followed previously.